apps/nextra/pages/en/build/smart-contracts/book/variables.mdx

```diff
@@ -619,6 +619,48 @@ script {
 
 For more details, see [Move references](references.mdx).
 
+### Compound Assignments
+
+_Since language version 2.1_
+
+Move also supports compound assignment operators.
+
+| Syntax | Description                                                 |
+| ------ | ----------------------------------------------------------  |
+| `+=`   | Performs addition and update the left-hand value            |
```

Comment:
update --> updates

You might also want to be extremely explicit somewhere about the de-sugaring:

It's not good practice to have side-effects in the middle of an expression referring to a storage location to be modified, but to understand the behavior of such code, it may be useful to know that
`e1 += e2` is equivalent to `let tmp = e2; let lhs = &mut (e1); *lhs = *lhs + tmp`.
(And similar for other operations.)

Reply:
I ran a linter on the Aptos framework and the over 800 on-chain packages and found no single use case of LHS with side effects. So perhaps not bother the reader with unnecessary details. And also I think the current description is accurate.

Reply:
Well, it can go in the reference book if we ever write it.